data/human/news.txt

```diff
@@ -2150,3 +2150,164 @@ news "graffiti on spaceport wall"
 			"Veni, vidi, hyperjumped away."
 			"Those who dance will be called fools by those who cannot hear the music."
 			"Better half a sentence, "
+
+news "High Tension Situation"
+	location
+		attribute "ccor"
+	name
+		"Standoff"
+	message
+		word
+			"CCOR Guerrillas "
+			"Guerrillas in CCOR colors "
+		word
+			"are "
+		word
+			"distributing pamphlets, "
+			"delivering aid, "
+			"rallying against the local authorities, "
+			"supporting local protests, "
+		word
+			"when "
+		word
+			"planetary authorities "
+			"local pirates "
+		word
+			"accost "
+			"confront "
+		word
+			"them and demand that they "
+		word
+			"disperse. "
+			"abandon their objective. "
+			"come quietly. "
+		word
+			"The Guerrillas "
+		word
+			"bluster "
+			"saber-rattle "
+		word
+			"for a bit in response "
+		word
+			"but it all ends with them dispersing."
+			"and their opposition backs down."
```

Comment:
```suggestion
	name
		"Standoff"
	message
		word
			"On the corner"
			"By a landing pad"
			"At the entrance to the spaceport market, "
		word
			"some CCOR guerrillas are "
			"there's a group of people in CCOR colors "
		word
			"distributing pamphlets"
			"handing out literature"
			"delivering aid"
			"rallying against the local authorities"
			"supporting local protests"
		word
			". "
		word
			"When "
		word
			"the planetary authorities "
			"a gang of local pirates "
		word
			"accost "
			"confront "
		word
			"them and demand that they "
		word
			"disperse"
			"make themselves scarce"
			"abandon their objective"
			"come quietly"
		word
			", "
		word
			"the guerrillas "
			"the group "
		word
			"bluster at them "
			"saber-rattle "
		word
			"in response "
		word
			", but they end up dispersing."
			", and the two groups back down."
```
Should be the same as the previous suggestion

Reply:
Thank you.